python_orocos_kdl/doc/conf.py

```diff
@@ -38,16 +40,16 @@
 
 # General information about the project.
 project = u'Orocos KDL python bindings'
-copyright = u'2012, Ruben Smits'
+copyright = u'2020, Ruben Smits'
```

Comment:
I am not a lawyer, but I think the year in copyright statements is the year of the first publication (or a range of years, like 2012-2020). I would not update it to 2020 or later here and elsewhere.

Seems like many copyright statements have been updated in https://github.com/orocos/orocos_kinematics_dynamics/commit/61f457a8a759c8e1f8a491dbbc9ca97eebb886bf?

Reference:
- https://stackoverflow.com/questions/2390230/do-copyright-dates-need-to-be-updated

Reply:
I checked the entire repo for changes since the 1.4 release. I updated this in #298